gallery/src/components/NxCombobox/NxComboboxPredeterminedListExample.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useCallback, useRef, useState } from 'react';
+import { filter, map, prepend, range } from 'ramda';
+import { NxCombobox, DataItem, NxFormGroup }
+  from '@sonatype/react-shared-components';
+
+const array:string[] = ['Alabama', 'Alaska', 'Arizona', 'California', 'Colorado', 'Connecticut',
```

Comment:
`array` is a very vague variable name, lets call it `states`

Reply:
Updated here d0e442dc197dbeaced5824330a1d5864955e5657.